testsuite/MDAnalysisTests/analysis/test_hbonds.py

```diff
@@ -31,8 +31,12 @@
 
 import itertools
 import warnings
+try:
```

Comment:
Oh, I didn't think about this. Then we should probably use [six.StringIO](https://pythonhosted.org/six/#six.StringIO): something like
```python
from six import StringIO
```